hips/hip-0011.md

```diff
@@ -0,0 +1,347 @@
+---
+hip: 0011
+title: "CRD Handling in Helm 3"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2021-03-26"
+type: "informational"
+status: "draft"
```

Comment:
Typically "informational" HIPs are considered final after approval, as they're meant to serve as information to the general Helm community, and it's not really considered a living document compared to "feature" or "process" HIPs. Do you expect there to be further changes to this document or would you consider this as a final draft?

Do keep in mind that newer HIPs can replace or deprecate older HIPs, so if we change how CRDs are handled later on we can mark this one as superseded.

Reply:
The intention was to document why Helm 3 works the way it does. So it is informational IMO. If a new thing happens for Helm 4, I envision a new HIP for that.

The main thing I was hoping to accomplish is to give people a substantial reference to understand what we have already done and why.

Reply:
@bacongobbler @technosophos I think the status can be set to `final`. Is that the only change being noted here?

Reply:
yes

Reply:
@bacongobbler are we good to merge at this point? You still have a nay vote on this PR.